packages/server/__tests__/api/grants.test.js

```diff
@@ -312,8 +312,8 @@ describe('`/api/grants` endpoint', () => {
             it('includes this user\'s own agency when it is interested in the grant', async () => {
                 expect(json.some((r) => r.agency_id === agencies.own)).to.equal(true);
             });
-            it('excludes a subagency of this user\'s own agency when the subagency is interested in the grant', async () => {
-                expect(json.find((r) => r.agency_id === agencies.ownSub)).to.be.undefined;
+            it('excludes an agency outside  of this user\'s own tenant', async () => {
+                expect(json.find((r) => r.agency_id === 0)).to.be.undefined;
```

Comment:
Is it guaranteed at this point that agency 0 is interested in some grant so we know that undefined here means it was actually filtered out vs. it just doesn't exist?

Reply:
Yeah it is setup in the db seed run before the tests https://github.com/usdigitalresponse/usdr-gost/blob/tenant-scoping-apis2/packages/server/seeds/dev/ref/grants.js#L119